src/components/scenes/RampCreateScene.tsx

```diff
@@ -0,0 +1,1088 @@
+        }
+      />
+    </>
+  )
+}
+
+// Export separate components for buy and sell routes
+export const RampCreateBuyScene = (
+  props: BuySellTabSceneProps<'pluginListBuy'>
+): React.ReactElement => <RampCreateScene {...props} direction="buy" />
+export const RampCreateSellScene = (
+  props: BuySellTabSceneProps<'pluginListSell'>
+): React.ReactElement => <RampCreateScene {...props} direction="sell" />
+
+const getStyles = cacheStyles((theme: ReturnType<typeof useTheme>) => ({
+  flagIconLarge: {
+    width: theme.rem(1.5),
+    height: theme.rem(1.5),
+    borderRadius: theme.rem(0.75)
+  },
+  flagIconSmall: {
+    width: theme.rem(1),
+    height: theme.rem(1),
+    borderRadius: theme.rem(0.75)
+  },
+  inputRowView: {
+    flexDirection: 'row',
+    alignItems: 'flex-start',
+    gap: theme.rem(1),
+    margin: theme.rem(0.5)
+  },
+  walletNameMaxRowView: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    paddingHorizontal: theme.rem(0.5),
+    paddingVertical: theme.rem(0.5)
+  },
+  walletNameText: {
+    padding: theme.rem(0.25),
+    color: theme.secondaryText,
+    fontFamily: theme.fontFaceDefault,
+    fontSize: theme.rem(0.75),
+    marginRight: theme.rem(0.5)
+  },
+  maxButton: {
+    padding: theme.rem(0.25),
+    borderRadius: theme.rem(0.5),
+    borderColor: theme.escapeButtonText
+  },
+  maxButtonText: {
+    color: theme.escapeButtonText,
+    fontFamily: theme.fontFaceDefault,
+    fontSize: theme.rem(0.75),
+    includeFontPadding: false as const
+  },
+  exchangeRateTitle: {
+    fontSize: theme.rem(1),
+    color: theme.primaryText,
+    textAlign: 'center' as const,
+    marginBottom: theme.rem(0.5),
+    marginTop: theme.rem(1)
+  },
+  exchangeRateValueText: {
+    fontSize: theme.rem(1.125),
+    fontWeight: 'bold' as const,
+    color: theme.primaryText,
+    textAlign: 'center' as const,
+    marginBottom: theme.rem(1)
+  },
+  stepsCard: {
+    marginHorizontal: theme.rem(0.5),
+    marginVertical: theme.rem(0.5),
+    padding: theme.rem(1),
+    backgroundColor: theme.cardBaseColor,
+    borderRadius: theme.rem(0.5),
+    borderWidth: theme.thinLineWidth,
+    borderColor: theme.cardBorderColor
+  },
+  stepRow: {
+    flexDirection: 'row' as const,
+    alignItems: 'flex-start' as const,
+    marginVertical: theme.rem(0.25),
+    gap: theme.rem(0.5)
+  },
+  stepNumberText: {
+    fontWeight: '600' as const,
+    minWidth: theme.rem(1.25)
+  },
+  stepText: {
+    flex: 1
+  },
+  regionButton: {
+    flexDirection: 'row' as const,
+    alignItems: 'center' as const,
+    backgroundColor: theme.cardBaseColor,
+    borderRadius: theme.rem(0.5),
+    margin: theme.rem(0.5),
+    padding: theme.rem(1),
+    borderWidth: theme.thinLineWidth,
+    borderColor: theme.cardBorderColor,
+    gap: theme.rem(0.5)
+  },
+  regionButtonText: {
+    flexShrink: 1,
+    color: theme.primaryText,
+    fontSize: theme.rem(1.1),
```

Comment:
For 99% of situations we should be using rem values divisible by 0.25. Not
really necessary to change because I already made a task to just do away with
this entire variant of the scene but just FYI